### DISCUSSION TITLE
First Stab at Documentation

### DISCUSSION THREAD

**Discussion by mitsuhiko** [astral-sh/rye#164] [Announcements]
With the help of @cnpryer I added the beginning of better documentation for Rye. At this point I think it's in a good enough state structurally. I have also started cleaning the readme as most of that information now when to the documentation instead.

* [User Guide](https://rye-up.com/guide/): main documentation
* [Philosophy and Vision](https://rye-up.com/philosophy/): more details about what I have in mind with the project

**Comment by schuerg** [comment]
The links seem to be broken. Here are the correct ones:

- **Rye User Guide**: https://rye-up.com/guide/
- **Rye Philosophy and Vision**: https://rye-up.com/philosophy/

**Reply by charliermarsh** [reply]
Thanks, I've updated the links in the parent post.
